cmds/modules/networkd/main.go

```diff
@@ -126,6 +128,17 @@ func action(cli *cli.Context) error {
 		return errors.Wrap(err, "failed to start yggdrasil")
 	}
 
+	log.Debug().Msg("starting mycelium")
```

Comment:
I think this and ygg code can be extracted into their own functions and same for the DMZ